src/components/ProjectDialog.tsx

```diff
@@ -89,16 +95,20 @@ const ProjectDialog: React.FC<ProjectDialogProps> = ({
               </Carousel>
              <Grid container spacing={3} sx={{marginBottom: '32px', marginTop: '32px'}}>
                    <Grid item xs={12} md={6}> 
-                      <Typography
-                      sx={{color: '#445E77', letterSpacing: '-0.5px', marginBottom: '24px'}}
-                      >{project.description}</Typography>
+                      <Box
+                          sx={{color: '#445E77', letterSpacing: '-0.5px', marginBottom: '24px', fontFamily: "'Figtree Variable',sans-serif"}}
+                          >
+                             <ReactMarkdown components={{ a: ({ node, ...props }) => modifyLinkTarget(props.href, props.title, props.children) }}>
```

Comment:
Modifying markdown internals can have serious security issues. I think this should be moved to a separate pullrequest and need tests before accepting it.

Reply:
Moved to a separate pull request - https://github.com/oasisprotocol/playground/pull/37